internal/app/machined/pkg/system/runner/containerd/containerd.go

```diff
@@ -115,8 +116,17 @@ func (c *containerdRunner) Close() error {
 func (c *containerdRunner) Run(eventSink events.Recorder) error {
```

Comment:
nit: we can have a named return: `(err error)` instead.

Reply:
we could, but I'm not sure there's any benefit to that since most all of the returns are wrapping the error ( `fmt.Errorf(xxx, err)` ) and not actually returning `err`